docs/server-side/v2/api-reference/analytics/overview.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Overview
+nav: 3.61
+---
+
+## Analytics APIs
+
+Analytics APIs are used to fetch events with an HTTP request. Currently, we support retrieving [track events](/server-side/v2/api-reference/analytics/track-events).
```

Comment:
this statement is not true since we also support recording events?

Reply:
I don't want to say recording events here since that doc won't be published. I'll reframe this sentence.